pkg/Microsoft.Private.CoreFx.NETCoreApp/Microsoft.Private.CoreFx.NETCoreApp.pkgproj

```diff
@@ -42,30 +42,32 @@
     <NETStandardVersion>2.0</NETStandardVersion>
   </PropertyGroup>
 
+  <Import Condition="'$(PackageTargetRuntime)' == ''" Project="$(RefBinDir)\*.props" />
   <ItemGroup Condition="'$(PackageTargetRuntime)' == ''">
-    <ProjectReference Include="@(Project)" />
-
     <!-- Include refs -->
-    <RefFile Include="$(RefBinDir)/*.*">
-      <TargetPath>ref/$(TargetFramework)</TargetPath>
-    </RefFile>
-    <File Include="@(RefFile)" />
+    <File Include="@(FileToPackage)">
```

Comment:
I should eventually.  We already have lots of duplication here so I'm not tackling that just yet.